addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -321,9 +335,7 @@
                             <t t-if="journal_type == 'purchase'">
                                 <field name="entries_count" invisible="1"/>
                                 <t t-if="record.entries_count.raw_value > 0">
-                                    <button class="btn btn-primary o_button_upload_document oe_kanban_action_button" journal_type="purchase" groups="account.group_account_invoice">
-                                        <span>Upload</span>
-                                    </button>
+                                    <t t-call="AccountKanbanUploaderButton"/>
```

Comment:
Aren't you missing the `groups="account.group_account_invoice"`?

Reply:
🤔 I need to check the drag and drop feature too